### PR TITLE
fix: changing input type as button in contact us page

### PR DESCRIPTION
**Pull request by Manoj0Marmat**

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@pankajan05  hello sir plz review this.

**Comment by Manoj0Marmat** [comment]
@pankajan05 this bug fix it on contact us page. i'll fix it on home page too

File: contact.html [hunk 1]
```diff
@@ -223,7 +223,7 @@ <h1 class="wow fadeInUp" data-wow-delay="0.3s">Get in touch</h1>
                             <div class="sent-message">Your message has been sent. Thank you!</div>
                         </div>
 						<div class="col-md-offset-3 col-md-6 col-sm-offset-2 col-sm-8">
-							<input name="submit" type="submit" class="form-control submit" id="submit" value="SEND MESSAGE">
+							<input name="submit" type="button" class="form-control submit" id="submit" value="SEND MESSAGE">
```
Comment (pankajan05) [inline contact.html:226]: The type must be "submit".  The real issue in the button that is not visible as the button to the users. You need to work on the CSS only.

Comment (pankajan05) [reply contact.html:226]: image

The button must be look like this and it must have the hover effect.

Comment (Manoj0Marmat) [reply contact.html:226]: @pankajan05 okay

Comment (Manoj0Marmat) [reply contact.html:226]: @pankajan05 assign this bug to me i'll fix it

Comment (pankajan05) [reply contact.html:226]: @Manoj0Marmat I really appreciate your enthusiasm. Please contact this Email : lilan@elzian.com

Comment (Manoj0Marmat) [reply contact.html:226]: @pankajan05 i fix it

Comment (Manoj0Marmat) [reply contact.html:226]: @pankajan05 somehow the background, colour and hover:background not apply properly that's why submit massage button messed up

**Comment by Manoj0Marmat** [comment]
@pankajan05 fixed somehow the background, colour and hover:background not apply properly that's why submit massage button messed up

**Comment by Manoj0Marmat** [comment]
@pankajan05 i create new pull request check

**Comment by pankajan05** [comment]
Your work is checked. good job @Manoj0Marmat and Thank you @Manoj0Marmat .
Please contact this email : lilan@elzian.com

**Pull request closed without merging**